js/models/listing/Item.js

```diff
@@ -90,13 +94,32 @@ export default class extends BaseModel {
 
     if (!attrs.images.length) {
       addError('images', app.polyglot.t('itemModelErrors.imageRequired'));
-    } else if (attrs.images.length > this.maxImages) {
-      addError('images', `The number of images cannot exceed ${this.maxImages}`);
+    } else if (attrs.images.length > max.images) {
+      addError('images', `The number of images cannot exceed ${max.images}`);
+    }
+
+    if (attrs.tags) {
+      if (is.array(attrs.tags)) {
+        if (attrs.tags.length > max.tags) {
+          addError('tags',
+            app.polyglot.t('itemModelErrors.tooManyTags', { maxTags: max.tags }));
+        }
+
+        attrs.tags.forEach((tag, index) => {
+          if (is.not.string(tag)) {
+            addError('tags', `Tag at index ${index} is not a string and should be.`);
```

Comment:
As usual, if you're seeing an untranslated model validation, it's because the UI has a validation mechanism in place (e.g. maxLength) that will never allow the user to enter bad data. The validation is in the model for data integrity purposes, in case the model is set programatically.